src/test/resources/application-test.yml

```diff
@@ -8,8 +8,5 @@ scheduling:
     pause-delay: 30000
 retry:
```

Comment:
d: You removed the retry config completely, so i think the complete retry block can be removed here?

Reply:
good catch, forgot about this one. also removed spring-retry from gradle build while on it